README.md

```diff
@@ -128,19 +128,15 @@ see the [osbuild-getting-started project](https://github.com/osbuild/osbuild-get
 
 API slice definitions are programmatically generated using the [@rtk-query/codegen-openapi](https://redux-toolkit.js.org/rtk-query/usage/code-generation) package.
 
-OpenAPI schema for the endpoints are stored in `/api/schema`. Their
-corresponding configuration files are stored in `/api/config`. Each endpoint
-has a corresponding empty API slice and generated API slice which are stored in
-`/src/store`.
+The OpenAPI schema are imported during code generation. OpenAPI configuration files are
+stored in `/api/config`. Each endpoint has a corresponding empty API slice and generated API
+slice which are stored in `/src/store`.
 
 ### Add a new API schema
 
 For a hypothetical API called foobar
 
-1. Download the foobar API OpenAPI json or yaml representation under
-`api/schema/foobar.json`
-
-2. Create a new "empty" API file under `src/store/emptyFoobarApi.ts` that has following
+1. Create a new "empty" API file under `src/store/emptyFoobarApi.ts` that has following
```

Comment:
One last nit pick - can you please update the list numbering??

<img width="454" height="962" alt="image" src="https://github.com/user-attachments/assets/f8429923-82ea-4e70-a7d2-1696a0355d3d" />

Reply:
Whoops, got one but missed the other 🤦‍♂️